src/floats.jl

```diff
@@ -468,8 +480,7 @@ unalias_bigfloat(::Type{T}, x) where {T} = T(x)
     else
         x = v / exp10(-exp)
     end
-    # See https://github.com/JuliaData/CSV.jl/issues/938
-    return neg ? T(-x) : unalias_bigfloat(T, x)
+    return convert_and_apply_neg(T, x, neg)
```

Comment:
I don't really see how this avoids an extra copy. Previously, the `-x` made one "copy", now the `unalias_bigfloat` makes one copy.

Reply:
Hmmmm, yeah, I think I was confusing the path where we we did something along the lines of `y = T(x)`, then `return neg ? -y : y`, which would be an extra allocation. But in any case, it feels more straightforward to not rely on knowing that `-x` makes a copy and just handle it explicitly.